app/src/gecko/java/org/mozilla/focus/web/GeckoErrorPage.kt

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.web
+
+import android.content.Context
+import android.support.v4.util.ArrayMap
+import android.support.v4.util.Pair
+import android.webkit.WebViewClient
+import org.mozilla.focus.R
+import org.mozilla.focus.utils.HtmlLoader
+import org.mozilla.geckoview.GeckoSession
+import java.util.HashMap
+
+object GeckoErrorPage {
```

Comment:
I started to create a separate library for the error pages (so that UX can design them once for all our products). Let's see if we can mix our work together.